coalib/parsing/CliParsing.py

```diff
@@ -80,13 +80,13 @@ def parse_custom_settings(sections,
     :param line_parser:          The LineParser to use.
```

Comment:
for the commit message: instead of ``shouldn't``, i think you should use ``doesn't``